### ISSUE TITLE
[BUG][types] index.d.ts doesn't contain valid types

### ISSUE THREAD

**Issue by pikax** [LouisMazel/maz-ui#141]
**Describe the bug**

Current [index.d.ts](https://cdn.jsdelivr.net/npm/maz-ui@2.3.5/lib/index.d.ts) contains invalid typescript exports

**To Reproduce**
1. Open url: https://cdn.jsdelivr.net/npm/maz-ui@2.3.5/lib/index.d.ts

**Expected behavior**
To have valid types

**Comment by LouisMazel** [comment]
Hi @pikax,

This file type is auto generated by Typescript. He is just here for the support of typescript. It's not a really a type declaration file. 

What do you expect in this file ?

**Comment by pikax** [comment]
I might have pointed the wrong file.

I updated to the latest `2.3.5` from `2.0.2` and my typescript import just stopped working
```ts
import { MazPhoneNumberInput } from "maz-ui";
```


I'm getting this error building with `quasar`
```bash

C:/dev/app/node_modules/maz-ui/lib/index.d.ts
ERROR in C:/dev/app/node_modules/maz-ui/lib/index.d.ts(2,1):
TS2309: An export assignment cannot be used in a module with other exported elements.

 App · ⚠️  [FAIL] Build failed with 1 error. Check log above.
```

This happens because the file pointed by the [package.json](https://github.com/LouisMazel/maz-ui/blob/master/package.json#L10) file is not valid `module` file, on the [2.0.2](https://github.com/LouisMazel/maz-ui/blob/v2.0.2/package.json#L114) it was pointing to a [different file](https://github.com/LouisMazel/maz-ui/blob/v2.0.2/index.d.ts)

Checked the releases but there's no mention of this change.


Also checked the other files and they are also invalid:
![image](https://user-images.githubusercontent.com/4620458/105827107-7dda9e80-5fb9-11eb-808c-688bfaec011b.png)

> This file type is auto generated by Typescript. He is just here for the support of typescript. It's not a really a type declaration file.

As seen in my screenshot above not even those files are valid module declaration,  based on `Typescript@4.1.2`

**Comment by LouisMazel** [comment]
Ok I see.
I keep you in touch

**Comment by LouisMazel** [comment]
You can follow this PR #166

**Comment by LouisMazel** [comment]
@seyfer 

Solved [v2.3.8](https://github.com/LouisMazel/maz-ui/releases/tag/v2.3.8)

**Comment by seyfer** [comment]
@LouisMazel great!!! thank you a lot
